File: src/test/java/fizzBuzz/FizzBuzzerTest.java
package fizzBuzz;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class FizzBuzzerTest {

    @Test
    public void name() {
        assertThat(true).as("This is not a good test").isEqualTo(false);
    }
}

File: src/test/kotlin/kotlinCoreTDDTests/fizzBuzz/FizzBuzzerTest.kt
package kotlinCoreTDDTests.fizzBuzz

import kotlinCoreTDD.fizzBuzz.FizzBuzzer
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class FizzBuzzerTest {

    @Test
    fun fizzBuzz_returns_Buzz_for_3() {
        assertThat(FizzBuzzer().execute(3)).isEqualTo("Buzz")
    }
}